### PR TITLE
Week 2 improve

### PR DESCRIPTION
**Pull request by oolusina**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -20,22 +20,36 @@
...
+    PreparedQuery results = datastore.prepare(
+        new Query("Comment").addSort("timestamp", SortDirection.DESCENDING)
+    );
 
-    //Responding with json-converted ArrayList
-    String json = convertToJson(comments);
+    ArrayList<String> comments = new ArrayList<String>();
+    for (Entity entity : results.asIterable()) {
+      String content = (String) entity.getProperty("content");
+      if (comments.size() < 8) comments.add(content);
```
Comment (abjindal-sps) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: why is 8 hard coded here? what does it refer to?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -27,23 +33,14 @@ function addRandomQuote() {
       'Give up? Give up?! The day may come when we’ll give up on fruitless searches after a mere 11 minutes, but that day is NOT today! The day may come when our favorite reptile may be lost from our memories and his enduring love of mushrooms forgotten, but that day is not today! Today we search! We will search for him in the streets, we will search for him in the trenches, we will search for him in the alleys and the mini-malls and the cul-de-sacs of this fair land. We will search for him in the multi-level car parks and municipal recreation facilities. And we few, we happy few, we small band of brothers - and girl from across the street. We shall not cease ’til he is found! - Ferb (Phineas and Ferb)', 
       'I’m just the keeper of secrets. Like the Vatican. - Jamal (On My Block)'];
 
-  // Pick a random quote.
+  // Picks random quote
   const quote = quotes[Math.floor(Math.random() * quotes.length)];
```
Comment (abjindal-sps) [inline portfolio/src/main/webapp/script.js:37]: ideally, we should use a int cast here.

**Review by abjindal-sps — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (oolusina) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: Limits number of comments shown to 8 as stated in the last commit for 'week 2 store'. I struggled to find a way to update this with a button, so I will be looking more into it during this week.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (oolusina) [reply portfolio/src/main/webapp/script.js:37]: Why should I int cast? Also, I think this code was provided by SPS.

Comment (abjindal-sps) [reply portfolio/src/main/webapp/script.js:37]: It seems an implicit way to typecast the underlying floating point value to an int, we generally prefer to have explicit casts.

**Pull request merged**